gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb

```diff
@@ -1,913 +1,900 @@
 {
```

Comment:
<p>Rapid Eval SDK GA&nbsp; ==&gt; "Gen AI Eval Service SDK".  Throughout. </p><p> </p><p>Please update the hyperlinks if you changed the filename.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>